### ISSUE TITLE
Bunker doesn't remember permission and the "Fully trust" option

### ISSUE THREAD

**Issue by dtonon** [greenart7c3/Amber#160]
I'm testing Amber with https://git.fiatjaf.com/song and it keep asking the permission for kind:30618 even if this kind is already present in the permission list _and_ I flag the "Fully trust" option.
Actually they seem two distinct bugs, but they probably could be correlated.

**Comment by greenart7c3** [comment]
How do I test this?

**Comment by dtonon** [comment]
I don't know, but I don't think it's specific to song, maybe is it a general bug of the bunker part?

**Comment by greenart7c3** [comment]
I think i managed to fix it. I'll make a release later so you can test

**Comment by dtonon** [comment]
Great!

**Comment by greenart7c3** [comment]
Just released version 3.0.3

**Comment by dtonon** [comment]
Unfortunately, it gives me the same behavior.

**Comment by greenart7c3** [comment]
Can you check in the settings if you are in the version 3.0.3?
I forgot to pull the changes before compiling the release

**Comment by dtonon** [comment]
Yes, already checked before giving the update.

**Comment by greenart7c3** [comment]
Does a new notification appear every time for the event after you approve it?

**Comment by dtonon** [comment]
Yes.
I noticed that when I select "Fully trust" and confirm the action, the view refresh and for a few instants I can see that the checkbox move back to the second option  ("Manually approve...").

**Comment by greenart7c3** [comment]
Are you using a custom relay or the default relay.nsec.app?

**Comment by greenart7c3** [comment]
Can you make a video of whats happening? for me it just show the kind:30618 once

**Comment by dtonon** [comment]
I created a new bunker and now I'm getting 'response error: unauthorized', so I cannot reproduce the first correct push.

**Comment by greenart7c3** [comment]
When you have some time try out the version 3.0.5 and let me know if it fixed the issue

**Comment by greenart7c3** [comment]
> > relay.nsec.app + relay.damus.io
> 
> Neither of those works anymore with remote signing, am I correct in stating that? I just tried, neither works.
> 
> This works:
> 
> relay.nip46.com

Yes, relay.nsec.app is not working properly and damus has some rate limits
